services/chaospod_test.go

```diff
@@ -0,0 +1,1465 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package services_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	chaosapi "github.com/DataDog/chaos-controller/api"
+	chaosv1beta1 "github.com/DataDog/chaos-controller/api/v1beta1"
+	builderstest "github.com/DataDog/chaos-controller/builderstest"
```

Comment:
```suggestion
	"github.com/DataDog/chaos-controller/builderstest"
```

Reply:
I don't really know why, but my IDE (GoLand) forced me to use an alias. Maybe, because the package is dedicated for testing purpose (_test suffix).

Reply:
hmm, in that case, you can ignore this feedback